README.md

```diff
@@ -115,8 +115,10 @@ Tracing.
 #### Talks
 
 * [Bay Area Rust Meetup talk and Q&A][bay-rust-2018-03], March 2018
+* [RustConf 2019 - Tokio-Trace: Scoped, Structured, Async-Aware Diagnostics][rust-conf-2019-08], August 2019
```

Comment:
Personally, I'd prefer to have the link reflect the actual name of the library, rather than `tokio-trace`:
```suggestion
* [RustConf 2019 - Tracing: Scoped, Structured, Async-Aware Diagnostics][rust-conf-2019-08], August 2019
```
Or, we could just remove the title from the link. What do you think?

Reply:
Removing the title is a good idea because what `tracing` does is already described in the readme